helm/values.yaml

```diff
@@ -3,7 +3,7 @@ deployment:
   replicasCount: 3
   image: securesystemsengineering/connaisseur:v2.0.0
   helmHookImage: securesystemsengineering/connaisseur:helm-hook-v1.0
-  imagePullPolicy: Always
```

Comment:
Doesn't this break local installation after cloning via `make install` if the image wasn't build before?

Reply:
not sure why this is marked as a change, but `imagePullPolicy: Always` has always been the setting

Reply:
Yes and for good reason I think. Would revert this to not break installation flow